background/background.js

```diff
@@ -598,7 +564,7 @@ twpConfig.onReady(function () {
     }
 })
 
-twpConfig.onReady(function () {
+twpConfig.onReady(async () => {
```

Comment:
Any special reasons to use async here?

Reply:
This function includes big processes, so we can make it asynchronous to don't affect overall extension, also nothing blocks it to make asynchronous